assets/templates/layouts.settings-surveys.hbs

```diff
@@ -26,17 +26,20 @@
 <main role="main">
 
     <div class="toolbar">
-        <div class="fab">
-            <a href="settings-survey-add.html" class="button button-alpha button-fab">
-                <svg class="iconic">
-                  <use xlink:href="../../img/iconic-sprite.svg#plus"></use>
-                </svg>
-                <span class="hidden">Add</span>
-            </a>
-        </div>
     </div>
 
     <div class="full-col">
+        <div class="listing card">
+            <article class="listing-item">
```

Comment:
This pattern needs work on smaller screens, specifically how the button and heading/paragraph interact.

I am open to ideas but see `/assets/html/5_layouts/md-archive.html` for an existing example that could be duplicated here. Maybe we could implement FlexBox into this pattern?(FYI, FB is amazing and although the PL is not built on FB I suggest implementing it anywhere that you can. I have been implementing FB for a while now because it makes building complex patterns much easier.

Also `listing card` is a "generic" site-wide component so be careful not to negatively affect other components with style adjustments.

Reply:
@middle8media I have added a new card instead of above solution for this pattern, with flexbox :)